spec/spec_helper.rb

```diff
@@ -19,6 +19,8 @@ def fixtures_dir
 RSpec.configure do |c|
   c.include SpecHelpers
   c.extend SpecHelpers # makes fixtures_dir available for example groups too
+  # https://relishapp.com/rspec/rspec-core/docs/command-line/only-failures
+  c.example_status_persistence_file_path = 'spec/examples.txt'
```

Comment:
I feel this one is kind of in the developers' preference territory. I wouldn't really use it (it's not how I am used to run tests) and I would probably feel "bothered" by that file that always gets added in the dir. I guess others might feel the same.

If you really want this there, I don't mind. If no one has anything against it, feel free to ignore my comment.

Reply:
> I feel this one is kind of in the developers' preference territory.

totally agree!

>  would probably feel "bothered" by that file that always gets added in the dir

but even with this file added in .gitignore?

Reply:
I'm fine with this change if the file is in gitignore